### PR TITLE
C407670: Verify the item status language in the instances item list when session locale is "German" (folijet) (TaaS)

### PR DESCRIPTION
**Pull request by IlyaLyskavets**
[Test Rail](https://foliotest.testrail.io/index.php?/cases/view/407670)
allure: 
image

### CODE REVIEW

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+  after('Delete test data', () => {
+    cy.getAdminToken().then(() => {
+      Users.deleteViaApi(testData.user.userId);
+    });
+  });
+
+  it(
+    'C407670 Verify the item status language in the instances item list when session locale is "German" (folijet) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.folijet] },
```
Comment (dnovikau) [inline cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js:54]: Should be { tags: ['extendedPath', 'folijet'] },

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    () => {
+      TenantPane.selectTenant(TENANTS.LANGUAGE_AND_LOCALIZATION);
```
Comment (dnovikau) [inline cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js:56]: Should remove local settings back to default in 'after' hook as this changes will affect other tests

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js (hunk 1 above)
Comment (dnovikau) [inline cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js:54]: Shoul add 'nonParallel' to tags array to run the test in insolated scope

**Review by dnovikau — CHANGES_REQUESTED** [review]

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js (hunk 1 above)
Comment (IlyaLyskavets) [reply cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js]: changed

Comment (IlyaLyskavets) [reply cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js]: added

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js (hunk 2 above)
Comment (IlyaLyskavets) [reply cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js:65]: fixed

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+import Users from '../../../support/fragments/users/users';
+import { randomFourDigitNumber } from '../../../support/utils/stringTools';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import { Localization } from '../../../support/fragments/settings/tenant/general';
+import TemporarySessionLocale from '../../../support/fragments/settings/developer/session-locale/temporarySessionLocale';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+
+describe('Settings: Tenant', () => {
```
Comment (TetianaParanich) [inline cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js:14]: Please use discribes like describe('inventory', () => {
  describe('Item', () => {

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  const itemBarcode = Helper.getRandomBarcode();
+  const HoldingData = {
+    rowNumber: 1,
+    status: 'Verfügbar',
+    accordionHeaderLanguageValue: 'Bestand: Main Library',
+  };
+
+  before('Create test data', () => {
+    cy.getAdminToken().then(() => {
+      InventoryInstances.createInstanceViaApi(instanceTitle, itemBarcode);
```
Comment (TetianaParanich) [inline cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js:31]: The instance should has 1 holdings and 2 items

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js (hunk 3 above)
Comment (IlyaLyskavets) [reply cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js]: Done in a way of: describe('Settings', () => {
describe('Tenant', () => {

File: cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js (hunk 4 above)
Comment (IlyaLyskavets) [reply cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js]: As I can see in createInstanceViaApi method it indeed has 1 holding and 2 items that are created inside of an instance.
Also as test running I also see 2 items.
image

**Review by dnovikau — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**